File: program/convert-pound-to-kilogram/ConvertPoundToKilogram.cpp
#include <iostream>
using namespace std;

int main() {
    double pounds, kilograms;
    const double CONVERSION_FACTOR = 0.453592;

    // Get input from user
    cout << "Enter weight in pounds: ";
    cin >> pounds;

    // Convert pounds to kilograms
    kilograms = pounds * CONVERSION_FACTOR;

    // Display result
    cout << pounds << " pounds is equal to " << kilograms << " kilograms." << endl;

    return 0;
}

File: program/convert-pound-to-kilogram/convert_pound_to_kilogram.rs
use std::io;

fn main() {
    // Prompt the user to enter a weight in pounds
    println!("Enter a weight in pounds:");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let pounds: f64 = input.trim().parse().expect("Invalid input");

    // Convert the weight from pounds to kilograms
    let kilograms = pounds / 2.20462;

    // Display the result
    println!("{} pounds is {:.2} kilograms", pounds, kilograms);
}
